### DISCUSSION TITLE
It used to be named StackVM?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#220] [General]
Thanks to this commit I know this secret 😄 

https://github.com/objeck/objeck-lang/commit/1b8cd399d87eedc166cf7a73af6481ffd1929b4e

**Comment by objeck** [comment]
Yes, the project started out as a stacked based virtual machine (with IA-32 JIT support). After it was completed I back ported the Objeck  compiler prototype I worked on prior.
